packages/backend/src/extension.ts

```diff
@@ -158,6 +227,340 @@ export async function getConfigurationValue<T>(property: string): Promise<T | un
   return extensionApi.configuration.getConfiguration('bootc').get<T>(property);
 }
 
+async function getJSONMachineList(): Promise<MachineJSONListOutput> {
```

Comment:
For getting json machine list, etc, should we have this instead in a separate file so it's more maintainable (since extension.ts is getting huge now).